src/test/compile-fail/issue-3044.rs

```diff
@@ -15,6 +15,6 @@ fn main() {
     });
     //~^^ ERROR this function takes 2 parameters but 1 parameter was supplied
     //~| NOTE the following parameter types were expected
-    //~| NOTE expected 2 parameters
+    //~| NOTE _, _
```

Comment:
Sigh, this one got worse.  I wish I'd fixed the type variable names also.  Once they're fixed, this should look better